test/DynamoCoreWpfTests/DynamoViewModelUnitTest.cs

```diff
@@ -39,10 +40,17 @@ protected override DynamoModel GetModel()
         [SetUp]
         public override void Setup()
         {
+            Dispatcher.CurrentDispatcher.UnhandledException += CurrentDispatcher_UnhandledException;
             base.Setup();
             StartDynamo();
         }
 
+        private void CurrentDispatcher_UnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
+        {
+            e.Handled = true;
+            System.Console.WriteLine($"Unhandled exception thrown during test {TestContext.CurrentContext.Test.Name} with message : {e.Exception.Message}");
```

Comment:
Are you still not able to see this message log?

Reply:
I was able to see this in some cases